app/src/main/java/org/fossasia/susi/ai/chat/adapters/recycleradapters/SearchResultsAdapter.java

```diff
@@ -43,13 +44,25 @@ public RssViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     public void onBindViewHolder(final RssViewHolder holder, int position) {
         Datum datum = datumList.get(position);
         if (datum != null) {
-            if( datum.getTitle() != null && !datum.getTitle().isEmpty()) {
-                holder.titleTextView.setText(Html.fromHtml(datum.getTitle()));
+            if (TextUtils.isEmpty(datum.getLink())) {
+                if (TextUtils.isEmpty(datum.getTitle())) {
+                    holder.titleTextView.setText(Html.fromHtml(datum.getTitle()));
+                }
+                if (TextUtils.isEmpty(datum.getDescription())) {
+                    holder.descriptionTextView.setText(Html.fromHtml(datum.getDescription()));
+                }
+                holder.linkTextView.setText(datum.getLink());
+            } else {
+                if (TextUtils.isEmpty(datum.getName())) {
+                    holder.titleTextView.setText(Html.fromHtml(datum.getName()));
+                }
+                if (TextUtils.isEmpty(datum.getJerseyNumber())) {
+                    holder.descriptionTextView.setText(Html.fromHtml(datum.getJerseyNumber()));
+                }
+                if (TextUtils.isEmpty(datum.getPosition())) {
+                    holder.linkTextView.setText(Html.fromHtml(datum.getPosition()));
+                }
```

Comment:
Your logic is opposite in all these cases

Reply:
@iamareebjamal what does opposite logic mean?